gpgsync/common.py

```diff
@@ -20,6 +20,7 @@
 """
 import datetime, os, sys, re, platform, inspect, requests, socket
 from PyQt5 import QtCore, QtWidgets, QtGui
+import webbrowser
```

Comment:
FYI, there's no need for the webbrowser module. Qt5 has it's own way cross-platform way of opening links:
`QtGui.QDesktopServices.openUrl(QtCore.QUrl('http://www.example.com'))`